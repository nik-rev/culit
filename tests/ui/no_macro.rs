//! `custom_literal::int::km` does not exist, I expect this to be an error directly on `10km`

use culit::culit;

mod custom_literal {
    pub mod integer {}
}

#[culit]
fn main() {
    let a = 10km;
}
