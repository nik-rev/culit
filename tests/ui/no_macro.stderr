error[E0433]: failed to resolve: could not find `km` in `integer`
  --> tests/ui/no_macro.rs:11:13
   |
11 |     let a = 10km;
   |             ^^^^ could not find `km` in `integer`
